napari/_app_model/constants/_menus.py

```diff
@@ -17,6 +17,8 @@
 class MenuId(str, Enum):
     """Id representing a menu somewhere in napari."""
 
+    MENUBAR_WINDOW = 'napari/window'
```

Comment:
The test that is failing is:

```
      @pytest.mark.parametrize('menu_id', list(constants.MenuId))
      def test_build_qmodel_menu(qtbot, menu_id):
          """Test that we can build qmenus for all registered menu IDs"""
          app = get_app()
          menu = build_qmodel_menu(menu_id)
          qtbot.addWidget(menu)
  >       assert len(menu.actions()) >= len(app.menus.get_menu(menu_id))
  
  napari/_qt/_qapp_model/_tests/test_qapp_model_menus.py:13: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  self = <MenusRegistry at 0x7f99f441c760 (4 menus)>
  menu_id = <MenuId.MENUBAR_WINDOW: 'napari/window'>
  
      def get_menu(self, menu_id: MenuId) -> List[MenuOrSubmenu]:
          """Return menu items for `menu_id`."""
          # using method rather than __getitem__ so that subclasses can use arguments
  >       return self._menu_items[menu_id]
  E       KeyError: <MenuId.MENUBAR_WINDOW: 'napari/window'>
```

I can't figure out what else needs to be done to add `MenuId`'s?

Reply:
haven't looked to closely, but at first glance, it looks like you didn't bring over the call to `init_qactions` in https://github.com/napari/napari/pull/4826/files#diff-a668967bb40dcd4fe528babf5641ceb5f8c2bafb6b6e57e5ff6a4bce15e150a3R141 ... 

so your actions are never getting registered

Reply:
Thanks! Could you also tell me why `ACTIONS` in [`napari/_qt/menus/window_menu.py`](https://github.com/napari/napari/blob/main/napari/_qt/menus/window_menu.py) is an empty list?

Reply:
I cannot :joy:
looks like whatever was in there got cleared in https://github.com/napari/napari/pull/3190 ... but the empty list was left around 🤷